apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -125,19 +150,15 @@ def select_future_state(
     possible_states: List[torch.Tensor],
     q_values: torch.Tensor,
     N_s_a: torch.Tensor,
-    repetitions: Dict[int, list],
+    repetitions: Dict[int, int],
     c_1: float = 1.25,
     c_2: float = 19652,
     return_idx: bool = False,
 ) -> torch.Tensor:
     """Select the future state maximizing the upper confidence bound."""
     # q_values (1, K, 1)
     pi = torch.tensor(
-        [
-            len(repetitions[i])
-            for i in range(len(possible_states))
-            if i in repetitions
-        ]
+        [repetitions.get(i, 0) for i in range(len(possible_states))]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)